File: Games/Dot Wars/Assets/Scripts/Music.cs
using UnityEngine;
using System.Collections;

public class Music : MonoBehaviour {
	private static AudioSource instance;

	void Awake () {
		if (instance != null && instance != GetComponent<AudioSource>()) {
			Destroy(gameObject);
			return;
		}
		else {
			instance = GetComponent<AudioSource>();
		}
		DontDestroyOnLoad(gameObject);
	}
}

File: Games/Jump the Box/Assets/Scripts/Main/Music.js
#pragma strict

public class Music extends MonoBehaviour
{
    public static var instance : Music;
    public var Music : AudioClip;
    
    public static function GetInstance() : Music {
    return instance;
    }
 
    function Awake() 
    {
        if ( instance != null && instance != this ) 
        {
            Destroy( this.gameObject );
            return;
        } 
        else 
        {
            instance = this;
        }
 
        DontDestroyOnLoad( this.gameObject );
    }
    function OnLevelWasLoaded (level : int) {
		if (level == 1) {
			audio.volume = 0.3;
		}
		if (level == 0) {
			audio.volume = 1;
		}
	}
}